projects/igniteui-angular/migrations/update-13_0_0/index.ts

```diff
@@ -16,6 +16,8 @@ export default (): Rule => (host: Tree, context: SchematicContext) => {
     const GRIDS = ['IgxGridComponent', 'IgxTreeGridComponent', 'IgxHierarchicalGridComponent'];
     const TAGS = ['igx-grid', 'igx-tree-grid', 'igx-hierarchical-grid'];
     const tsFiles = update.tsFiles;
+    const SERVICES = ['IgxCsvExporterService', 'IgxExcelExporterService'];
+    const moduleTsFiles = update.moduleTsFiles;
```

Comment:
this could easily be `update.tsFiles.filter(x => x.endsWith('.module.ts'))`, no? Kind of no need to go through a separate getter that loops over the directories again IMO - doubt it's meaningfully slower, but it's still duplicate work :)